python/gui/auto_generated/qgisinterface.sip.in

```diff
@@ -672,69 +672,133 @@ Returns the Hide Deselected Layers action.
     virtual QAction *actionCheckQgisVersion() = 0;
     virtual QAction *actionAbout() = 0;
 
-    virtual QAction *actionCircle2Points()  = 0;
+
+ virtual QAction *actionCircle2Points() /Deprecated/;
```

Comment:
Maybe add a documentation to explain how to launch the maptools via iface (same for the others).

Reply:
These actions are used in some plugins (was asked for qgep/qwat, and after used by other tools)

Reply:
ok for you like this bf44303 ?

Reply:
Yes, thanks!